Client/Frontend/Browser/PasswordManager.swift

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Nit: I hope the next version of Xcode fixes these paste issues. Note that several of your new files here don't have these stars lined up.